python/lsst/obs/subaru/isr.py

```diff
@@ -300,10 +297,15 @@ def runDataRef(self, sensorRef):
             flatExposure = self.getIsrExposure(sensorRef, "flat")
             self.flatCorrection(ccdExposure, flatExposure)
 
+        if self.config.doDefect:
+            self.maskAndInterpDefect(ccdExposure, defects)
+        if doWriteDebug: ccdExposure.writeFits(outDir+"/isr7.fits")
```

Comment:
Oh, well, good, because I was *also* about to complain about the hard-coded `isr7.fits` filename. :-)